biomaj/schema_version.py

```diff
@@ -104,3 +107,74 @@ def migrate_pendings():
             print("Migration: %d bank production info updated" % updated)
 
         schema.update_one({'id': 1}, {'$set': {'version': installed_version}})
+
+    @staticmethod
+    def add_property(bank=None, prop=None, value=None, cfg=None):
+        """
+        Update properties field for banks.
+
+        :param bank: Bank name to update, default all
+        :type bank: str
+        :param prop: New property to add
+        :type prop: str
+        :param value: Property value, if cfg set, value taken
+                      from bank configuration cfg key
+        :type value: str
+        :param cfg: Bank configuration key value is taken from
+        :type cfg: str
+
+        :raise Exception: If not configuration file found
+        :returns: True/False
+        :rtype: bool
+        """
+        if BiomajConfig.global_config is None:
+            try:
+                BiomajConfig.load_config()
+            except Exception as err:
+                print("* SchemaVersion: Can't find config file: " + str(err))
+                return False
+        if prop is None:
+            print("Property key is required", file=sys.stderr)
+            return False
+
+        if MongoConnector.db is None:
+            MongoConnector(BiomajConfig.global_config.get('GENERAL', 'db.url'),
+                           BiomajConfig.global_config.get('GENERAL', 'db.name'))
+
+        schema = MongoConnector.db_schema
+        banks = MongoConnector.banks
+
+        schema_version = schema.find_one({'id': 1})
+        if schema_version is None:
+            schema_version = {'id': 1, 'version': '3.0.0'}
+            schema.insert(schema_version)
```

Comment:
as said above, I think schema should not be updated here, but if it was required, should be set to current biomaj package version, not an hard coded value.